.github/workflows/continuous-integration.yml

```diff
@@ -69,3 +75,65 @@ jobs:
         run: docker cp framework-jest-tests:/www/coverage ./
       - uses: codecov/codecov-action@v1
         if: always()
+
+  unstable-release:
+    runs-on: ubuntu-18.04
+    name: Unstable Release
+    needs: [integration-test]
+    # Only run on push to main branch
+    if: ${{ github.ref == 'refs/heads/main' && github.event == 'push' }}
+    steps:
+      - name: Checkout aries-framework-javascript
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: git config
+        run: |
+          git config user.name "@github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12.x'
+
+      - name: Install dependencies
+        run: yarn install --ignore-optional
+
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: yarn release patch --preRelease=unstable
+
+  stable-release:
+    runs-on: ubuntu-18.04
+    name: Stable Release
+    needs: [integration-test]
+    # Only run on workflow dispatch on main branch
+    if: ${{ github.ref == 'refs/heads/main' && github.event == 'workflow_dispatch' }}
+    steps:
+      - name: Checkout aries-framework-javascript
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: git config
+        run: |
+          git config user.name "@github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12.x'
+
+      - name: Install dependencies
+        run: yarn install --ignore-optional
+
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: yarn release ${{ github.event.inputs.releaseType }}
```

Comment:
It would nice to not have so much duplication in stable-release and unstable-release, but that's probably how it's done in GH Actions, right? :)

Reply:
It's already on my todo list!

I just discovered you can do `if` statement per step, not per job. So that'd mean only the last step would be different